app/boards/arm/bt60/bt60_v1.dts

```diff
@@ -6,15 +6,15 @@
 
 /dts-v1/;
 #include "bt60.dtsi"
+#include <layouts/common/60percent.dtsi>
 
 
 / {
     chosen {
-        zmk,kscan = &kscan0;
-        zmk,matrix-transform = &ansi_transform;
+        zmk,physical_layout = &layout_60_ansi;
     };
 
-    ansi_transform: keymap_transform_0 {
+    matrix_transform_60_ansi: matrix_transform_60_ansi {
```

Comment:
The boards in this pr that changed the matrix transform in the upstream keymap were the polarity works boards, which used funky preprocessor stuff to support multiple layouts, and the preonic. The Planck also provided multiple transforms for user selection. 

Polarity works have primarily been directing customers to their own editor and their hosted config repo redefines the whole board. Furthermore, I don't think the Preonic and Planck are especially popular boards for zmk. So the number of affected users would be limited, especially given only Planck users who change the default  transform would be affected.

The existing label could be retained using DT aliases I believe, I'll add that. I agree that this is a bit  of a blunt measure, but it only has to be done once.

Reply:
Im not keen on a breaking change, but I don't see any possible way to implement physical-layout without it being breaking currently on the btxx boards. 

That being said I have an idea that could resolve the issues. I can tweak the physical layout code to pick up the chosen transform if one is present, it's not an ideal solution as it won't handle multiple layouts but it will still work. Add in DT aliases to preserve old labels and then this pr won't be breaking

Reply:
I think in some of the boards Pete updated already, he did something more like

```dts
#include <layouts/whatever.dtsi>

&layout_whatever {
    transform = &my_keyboard_transform;
};
```

which would not require renaming the transforms. I also like it being more explicit rather than the layout defining a name for a transform which it then expects every keyboard using it to use.

Reply:
> > The boards in this pr that changed the matrix transform in the upstream keymap were the polarity works boards, which used funky preprocessor stuff to support multiple layouts
> 
> Fair enough for the people who are pointed at the PW repo, but the funky preprocessor stuff exacerbates the problem as it means [pretty much all BTXX keymaps have a `chosen` node with a `matrix-transform`](https://github.com/search?q=path%3A%2Fbt%5Cd%2B_.%2B%5C.keymap%2F+transform+NOT+path%3A%2Fapp%2F+NOT+%26reset&type=code) instead of just the nerds who got curious enough to examine the board definition.
> 
> No hard feelings if you still disagree and would rather just make the breaking change with no backwards compatibility, though. My comments aren't law, and there's already precedent; all 5-column Corne keymap builds were likely broken by the recent switch to `physical-layout`, and [that has a much greater impact](https://github.com/search?q=path%3A%2F%5C%2Fcorne.keymap%24%2F+transform+NOT+path%3Aapp+NOT+%26reset&type=code).

Yeah... I'm considering if there's any way to avoid this breakage, perhaps just *warning* when both are defined and falling back to the "use the assigned transform and ignore the layout" as a reasonable fix for this.